backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -234,7 +234,7 @@ trait StandardAsyncExecutionActor extends AsyncBackendJobExecutionActor with Sta
         |echo "${controlFileContent.trim}" > $globDirectory/$controlFileName
         |
         |# symlink all the files into the glob directory
-        |( ln -L ${globFile.value} $globDirectory 2> /dev/null ) || ( ln ${globFile.value} $globDirectory )
+        |( ln -L ${globFile.value} $globDirectory 2> /dev/null ) || ( ln ${globFile.value} $globDirectory 2> /dev/null)
```

Comment:
Curious why this error is being silenced?

If I end up with an empty array I generally want to know why. The prior `2> /dev/null` was only there because not all platforms support `-L`.

```wdl
Array[File] out = glob("oops_wrong_dir/*")
```

Produced previously:
```
ln: oops_wrong_dir/*: No such file or directory
```

In the case where one is using globs-for-optionals, then I guess there will be a "No such file or directory" generated, but a) it would only be in the stderr and b) would still help debug why one ended up with `Array[File] out = []`.

Reply:
With the disclaimer that this change is a stowaway not required for the rest of the PR to work, I thought  it could safely be assumed that if the array of files returned from the glob was empty it was because the glob did not match any files. An explicit message on stderr to that effect didn't seem to be adding value and looks scary in the logs. And due to the way the cwl.output.json is currently globbed we see that error message with nearly all CWL workflows, though we could probably handle cwl.output.json differently if we really wanted to keep this message for globs in general.

Reply:
If no other reviewers have a problem changing this warning then I guess commit it as is and see if we get anyone complaining. Ultimately I could see how the UX could be debated as an empty _accidental_ glob is due to PEBKAC, but an empty _intentional_ glob could maybe be used in some circumstances.

Also if you're reading this: right now `OutputEvaluatorSpec` is still in the wrong package/directory. Should be under `cromwell/backend`.